provisioning/requirements.yml

```diff
@@ -1,6 +1,7 @@
 - src: https://github.com/idi-ops/ansible-facts
   name: facts
 
-- src: https://github.com/gtirloni/ansible-role-nodejs
+- src: git+https://github.com/gtirloni/ansible-role-nodejs.git
```

Comment:
@cindyli I have merged the changes into https://github.com/idi-ops/ansible-nodejs, so feel free to revert the contents of requirements.yml as they were [before](https://raw.githubusercontent.com/GPII/universal/master/provisioning/requirements.yml).